R/pkg/inst/worker/daemon.R

```diff
@@ -32,6 +32,9 @@ inputCon <- socketConnection(
 
 SparkR:::doServerAuth(inputCon, Sys.getenv("SPARKR_WORKER_SECRET"))
 
+# Application-specific daemon initialization. Typical use is loading libraries.
+eval(parse(text = Sys.getenv("SPARKR_DAEMON_INIT")))
+
```

Comment:
To give you a sense of the cost of this, consider
```
> microbenchmark(NULL, times = 999999)
Unit: nanoseconds
 expr min lq     mean median uq   max  neval
 NULL   2  4 4.328575      4  4 13885 999999
> microbenchmark(eval(parse(text = "NULL")), times=9999)
Unit: microseconds
                       expr    min      lq    mean median     uq      max neval
 eval(parse(text = "NULL")) 31.489 34.0545 40.1457 35.938 38.496 9370.646  9999
> 
```
So, on my 2017 MacBook pro, it takes 34 microseconds.

For comparison, consider the 6 milliseconds we use up loading worker.R at every fork.